### PR TITLE
Flows Overhaul

### PR DESCRIPTION
**Pull request by erondpowell**
Redo Flows docs, beginning to end, making them more in-depth and more accessible.

- [x] Re-test all triggers and operations
- [x] Rewrite Triggers
- [x] Rewrite Operations
- [x] Rewrite Flows homepage
- [ ] ~~Add in-depth video content~~ (saving this for 2nd PR)

### CODE REVIEW

File: configuration/flows.md [hunk 1]
```diff
@@ -23,113 +23,356 @@ There is also dedicated API documentation on [Flows](/reference/system/flows) an
...
+13. Once created, you may need to re-edit your flow, toggle it to inactive, or delete it.
 
-## The Flow Object
+### Edit a Flow
+
+1. Navigate to the desired flow.
+2. Click <span mi btn muted>edit</span> in the flow page header and make reconfigurations as desired.
+3. Click <span mi btn>done</span> to confirm.
+
+### Toggle a Flow to Inactive
```
Comment (azrikahar) [inline configuration/flows.md:177]: Not sure do we want to change it to something such as `Toggle a Flow to Active or Inactive` or `Change the state of a Flow`. These are just very rough suggestions, more so about how we can communicate it as "we can make it active or inactive".

File: configuration/flows/operations.md [hunk 2]
```diff
@@ -1,170 +1,427 @@
...
 - **Payload** — Update Items in a Collection. To learn more, see [API > Items](/reference/items).
 - **Query** — Select Items to update with a query. To learn more, see [Filter Rules](/reference/filter-rules).
 
+**Payload**
+
+An array containing all items updated will be appended under its `<operationKey>`.
+
+**More Details**
+
+:::tip
```
Comment (azrikahar) [inline configuration/flows/operations.md:240]: `Create Data`, `Read Data` and `Delete Data` seems to be using `warning`, so perhaps this should also be a `warning` for consistency 👍

File: configuration/flows/triggers.md [hunk 3]
```diff
@@ -7,69 +7,171 @@ readTime: 3 min read
...
 ![Manual](https://cdn.directus.io/docs/v9/configuration/flows/triggers/triggers-20220603A/manual-20220602A.webp)
 
-This Trigger starts your Flow on a manual click of a button within the Directus App. When you use this Trigger, a
-**Flows** menu containing a button will appear in the Sidebar of the specified Collection Page(s) and/or Item Pages,
-based on your **Location** configuration.
+This Trigger starts your flow on a manual click of a button within the Directus App. When you use this trigger, a
+**Flows** menu containing a button will appear in the sidebar of the specified collection page(s) and/or its item pages.
+
+The manual trigger must take in item ID(s) to activate. From the collection page, the button will be _grayed out_ until
+you select some number of items. From the item page, the current item's ID is passed in automatically. These item IDs
```
Comment (azrikahar) [inline configuration/flows/triggers.md:162]: > From the collection page, the button will be _grayed out_ until
you select some number of items.

This is now slightly context-dependent after directus/directus#15977 added a boolean option "Collection Page" that indicates whether selection is required. Docs for it was added in #170, but I should've updated this as well 🤦‍♂️

**Comment by erondpowell** [comment]
@rijkvanzanten there are quite a few type-errors in this. 
Please kindly give me the "thumbs up" after you've made your review, then I'll give it the final pass.

File: configuration/flows.md [hunk 4]
```diff
@@ -23,113 +23,356 @@ There is also dedicated API documentation on [Flows](/reference/system/flows) an
...
+As you can see, the example above doesn't have any substantial data inside each key. In reality, there's going to be a
+lot of data and it will always be slightly different, based on your flow's unique configuration. During configuration
+and debugging, you'll need to use a tool like [The Log](#logs) to view your data chain and make sure each operation is
+accessing and generating data as you intended.
+
+:::tip
+
+In our examples, we are using generic _placeholders_ for operation keys, like `<operationKey>`, which might look funny
+to low-code users. In practice, operation keys will actually have unique and descriptive names, like `send_email_7538`.
+
```
Comment (emilyf) [inline configuration/flows.md:240]: Incredible updates here! The operationKey documentation is excellent and though it's clear you're using _placeholders_ in the above text this section is prob most difficult for a newer low code user to fully grasp...

The send_email_7538 example is what finally connected it for me and I've been cranking out on flows daily so I'm thinking about the newer user. There's a great image you already have on operations.md; what about including one here as well that has the key field highlighted and matches more closely to the tip?? Example below:
Screen Shot 2022-10-24 at 12 36 06 PM

**Pull request merged**